Procurement/ViewModel/LoginWindowViewModel.cs

```diff
@@ -185,6 +185,8 @@ public void Login(bool offline)
 
                 ApplicationState.SetDefaults();
 
+                ClientLogFileWatcher.Instance.Start();
```

Comment:
You have called start, but I can't see where you are calling stop. Is there an appropriate place to call stop?